code/modules/mob/living/carbon/metroid/metroid.dm

```diff
@@ -686,15 +686,6 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
 		..()
 		create_reagents(100)
 
-/obj/item/slime_extract/attack(mob/living/carbon/human/M as mob, mob/user as mob) //changing slime people colors
-		if(M.dna.mutantrace != "slime" || M != user) return
-
-		M.slime_color = _color
-		user <<"You absorb the core and your color shifts!"
-		M.update_mutantrace()
-		del(src) //Finally a tidy way to remove all the used cores lying about

```

Comment:
This seems like it could maintain functionality by just adding a new variable to slime extracts, something like an RGB list, var/colors